suite-native/module-home/src/screens/HomeDemoScreen.tsx

```diff
@@ -88,18 +90,18 @@ export const HomeDemoScreen = ({
                             size="small"
                             colorScheme="gray"
                             iconName="check"
-                            onPress={() => console.log('press icon button')}
+                            onPress={() => console.warn('press icon button')}
```

Comment:
no warn please, it doesn't make sense it this context

Reply:
https://github.com/trezor/trezor-suite/pull/5751/commits/1a3fc9a82c5d856cae3c4f42bd5f332ab8e43328